reference-architecture/aws-ansible/playbooks/validation.yaml

```diff
@@ -6,10 +6,10 @@
   pre_tasks:
       - name: set fact
         set_fact:
-           openshift_master_cluster_public_hostname: "{{ openshift_master_cluster_public_hostname }}"
+           openshift_master_cluster_public_hostname: "openshift-master.{{ public_hosted_zone }}"
       - name: set fact
         set_fact:
-           openshift_master_cluster_hostname: "{{ openshift_master_cluster_hostname }}"
+           openshift_master_cluster_hostname: "internal-openshift-master.{{ public_hosted_zone }}"
```

Comment:
Right so the play is to go to one of the masters and do a web call to ensure that they can hit the healthz page of the internal-only load balancer.  That way we can verify nothing crazy was set in regards to the cluster_hostname and the internal cluster vip is healthy. Making troubleshooting a bit easier if things weren't working as expected